tests/unit/plugins/modules/cloud/misc/test_proxmox_snap.py

```diff
@@ -0,0 +1,24 @@
+# Copyright: (c) 2019, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+import pytest
+
+proxmox = pytest.importorskip('proxmoxer')
```

Comment:
That won't help much for CI, since that package isn't around. It would be better if you could mock the ProxmoxAPI object.